aperturedb/transformers/image_properties.py

```diff
@@ -0,0 +1,32 @@
+from aperturedb.transformers.transformer import Transformer
+from aperturedb.Subscriptable import Subscriptable
+from PIL import Image
+import io
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class ImageProperties(Transformer):
+    """
+    This computes some image properties and adds them to the metadata.
+    """
+
+    def getitem(self, subscript):
+        x = self.data[subscript]
+        try:
+            # x is a transaction that has an add_image command and a blob
+            for ic in self._add_image_index:
+                # Compute the dynamic properties and apply them to metadata
+                x[0][ic]["AddImage"]["properties"]["adb_image_size"] = len(
+                    x[1][ic])
+
+                # Compute the image dimensions.
+                pil_image = Image.open(io.BytesIO(x[1][ic]))
+                x[0][ic]["AddImage"]["properties"]["adb_image_width"] = pil_image.width
```

Comment:
I like using these variables BUT I remember seeing COCO images get added with width, height which causes a duplicate. Do you know how to address that?

Reply:
I am not sure if this can be consolidated because of the down stream consumers of these properties.
Leaving them in right now. We should discuss this more.

Reply:
I created an issue for this